.github/actions/comment-pr-instructions/action.yml

```diff
@@ -35,15 +35,15 @@ runs:
             ```shell
             AUTHENTIK_IMAGE=ghcr.io/goauthentik/dev-server
             AUTHENTIK_TAG=${{ inputs.tag }}
-            AUTHENTIK_OUTPOSTS__CONTAINER_IMAGE_BASE=ghcr.io/goauthentik/dev-%(type)s:gh-%(build_hash)s
```

Comment:
This is on purpose as that placeholder is interpreted by authentik to use its matching build bash